py/bitbox02/setup.py

```diff
@@ -75,7 +75,7 @@ def find_version() -> str:
     install_requires=[
         "hidapi>=0.7.99.post21",
         "noiseprotocol>=0.3",
-        "protobuf>=3.7",
+        "protobuf>=3.21",
```

Comment:
AFAICT protobuf 3.20.x is able to parse both the old and the new format
(while <3.20 can only parse the old format, and >=3.21 can only parse the new format)

Hence, would it not be better to set this to `"protobuf>=3.20"` ?

Reply:
Interesting, how did you figure this out? The [release notes](https://github.com/protocolbuffers/protobuf/releases/) don't exactly give it away.

Anyway, we're happy to make a patch release that relaxes the constraint.

Fyi, protobuf>=3.20 requires Python 3.7, see also: https://github.com/bitcoin-core/HWI/issues/626

Reply:
> Interesting, how did you figure this out? The [release notes](https://github.com/protocolbuffers/protobuf/releases/) don't exactly give it away.

When you have new protobuf and old generated files, the runtime error says
` 1. Downgrade the protobuf package to 3.20.x or lower.`.
I manually tested a bunch of protobuf versions from PyPI around  that boundary, hoping some can parse both and to learn the behaviour.
I had tried reading the release notes first but could not make sense of them either :)

> Fyi, protobuf>=3.20 requires Python 3.7, see also: https://github.com/bitcoin-core/HWI/issues/626

That's fine. (in Electrum, we already bumped the minimum to Python 3.8)

Reply:
6.1.1 is out, please give it a try.